package.json

```diff
@@ -47,7 +47,6 @@
     "openapi3-ts": "3.2.0",
     "ramda": "0.28.0",
     "triple-beam": "1.3.0",
-    "typescript": "4.9.5",
```

Comment:
⚠️ Breaking: Typescript becomes an optional peer, needed for generating client only